package.json

```diff
@@ -1,41 +1,40 @@
 {
   "name": "tailor",
   "version": "5.0.0-rc.5",
-  "codename": "twill",
+  "private": true,
```

Comment:
Was there a specific reason to introduce these changes?

Reply:
sort package.json script 😉

Reply:
I got that part 🤡